sql/all_match_pairs_with_um.sql

```diff
@@ -19,7 +18,28 @@ WITH
   WHERE
     (ids.mag IS NOT NULL) AND ((type IS NULL)
         OR NOT (type IN ("dataset", "peer-review", "grant")))
-    ),
+  ),
+  s2_matches AS (
```

Comment:
This part of the query is setting us up to force s2 to match mag ids when s2 gives us this mapping